<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>184</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Name: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="uiProjectName"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Hamiltonian:</string>
   </property>
  </widget>
  <widget class="QListView" name="listHamilton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>181</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Potential Energy Surface:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="onRadio">
      <property name="text">
       <string>on</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListView" name="listPES"/>
    </item>
    <item row="0" column="1">
     <widget class="QRadioButton" name="offRadio">
      <property name="text">
       <string>off</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Type of calculation:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>274</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="RealRadio">
      <property name="text">
       <string>real - time propagation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="ImaginaryRadio">
      <property name="text">
       <string>imaginary - time propagation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="EigenstateRadio">
      <property name="text">
       <string>Eigenstate calculation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="fluxEigenstateRadio">
      <property name="text">
       <string>Thermal fluxe eigenstate calculation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Integrator:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>start time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uiStartTime"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>initialtime step:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uiInit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>end time: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uiEndTime"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>iteration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="uiIter"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Basis:</string>
   </property>
  </widget>
  <widget class="QTreeView" name="uiTree">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="uiDisplayTree">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>661</width>
     <height>391</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>670</y>
     <width>334</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="uiCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiSaveJob">
      <property name="text">
       <string>Save Job</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="uiStartCal">
      <property name="text">
       <string>Start calculation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
